client/stylesheets/_responsive.scss

```diff
@@ -123,78 +123,90 @@
 }
 
 @media (max-width: 650px) {
-    .homeCardLi {
-    	width: 38%;
-    }
-    .profileCardLi {
-    	width: 43% !important;
-    }
+    // .homeCardLi {
+    // 	width: 38%;
+    // }
+    // .profileCardLi {
+    // 	width: 43% !important;
+    // }
 }
 
 @media (max-width: 600px) {
-    .homeCardLi {
-    	width: 40%;
-    }
+    // .homeCardLi {
+    // 	width: 40%;
+    // }
 }
 
 @media (max-width: 580px) {
-    .profileCardLi {
-    	width: 60% !important;
-    }
+    // .profileCardLi {
+    // 	width: 60% !important;
+    // }
 }
 
 
 @media (max-width: 520px) {
-    .homeCardLi {
-    	width: 50%;
-    }
-    .profileCardLi {
-    	width: 60% !important;
-    }
+    // .homeCardLi {
+    // 	width: 50%;
+    // }
+    // .profileCardLi {
+    // 	width: 60% !important;
+    // }
 }
 
 @media (max-width: 500px) {
     .helpImgUl > li {
     	width: 40%;
     }
+
     .helpImgUl > li:last-child {
     	display: none !important;
     }
+
+    .faqdiv h3, .safetydiv h3, .reportdiv h3 {
+        margin-left: 10px;
+        margin-top: 30px;
+    }
+
+    .faqdiv ol, .safetydiv ol, .reportdiv ol {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Each selector in a comma sequence should be on its own single line